template/README.md

```diff
@@ -11,27 +11,97 @@ If you found this in `fredrikhgrelland/vagrant-hashistack`, you may be intereste
 
 Documentation on [parent repository](https://github.com/fredrikhgrelland/vagrant-hashistack#usage).
 
-## Customizing and using the vagrant box
+## Goals of this template
+This template aims to supply a developer with the tools necessary to do three things:
+- Build a docker image
+- Build a terraform module
+- Test and develop these two within a hashistack ecosystem
+
+To achieve this we utilise a vagrant-box built from [fredrikhgrelland/vagrant-hashistack](https://github.com/fredrikhgrelland/vagrant-hashistack/).
+
+## Where do I start?
+First of all I would recommend you read the section ["Using the template and box"](using-the-template-and-box) for a detailed look at the box's features and configurations. Below is a rough step-by-step going from a fresh template to a functioning terraform module. You can find example-code of everything below in [test_example/](test_example/).
+
+Step-by-step:
+1. Create a [Dockerfile](https://docs.docker.com/engine/reference/builder/) and put it in `docker/`.
+1. Use the box's startup playbooks to build and make the image available within the box. [Startup scheme](default-startup-scheme). [Example building image](test_example/dev/ansible/01_build_docker_image.yml).
```

Comment:
What is `(default-startup-scheme)`?

Reply:
that is a link to the Markdown title `## Defaul startup scheme`

Reply:
does it work? I thought smth like (#default-startup-scheme)

Reply:
This has been fixed, but you were right, it was written wrong!